plugins/github-actions/src/components/WorkflowRunDetails/WorkflowRunDetails.tsx

```diff
@@ -116,8 +116,8 @@ const JobListItem = ({
     <Accordion TransitionProps={{ unmountOnExit: true }} className={className}>
       <AccordionSummary
         expandIcon={<ExpandMoreIcon />}
-        aria-controls={`panel-${name}-content`}
-        id={`panel-${name}-header`}
+        aria-controls={`panel-${job.name}-content`}
+        id={`panel-${job.name}-header`}
```

Comment:
To be honest I have no idea what the intention of this is 😅. There's no element with a matching `panel-${name}-content` ID, and name here refers to `window.name`, so that's definitely a mistake. I think we can just remove all of these tbh because I don't see how they could be working? That goes for all 4 usages of this pattern in the repo, even the one that doesn't use the deprecated `window.name`.

Reply:
So the components must stay, but the attributes `id` and `aria-controls` can be removed as they are not being referenced anywhere else?

Reply:
Exactly!

Reply:
Awesome, I will take a look this afternoon. While a small change, this is a good opportunity because it can be intimidating to delete code from someone else's repo 😅.

Reply:
I've updated the two files, but you mentioned 
> That goes for all 4 usages of this pattern in the repo, even the one that doesn't use the deprecated window.name.

I searched the repo but couldn't find it being used anywhere else. Can you clarify what you meant here?